Makefile

```diff
@@ -54,6 +54,15 @@ build-docs-api: ## build Rest API docs
 build-docs-api: build
 	./scripts/build-rest-api-spec.sh
 
+.PHONY: clean
+clean: # Cleanup build directories (lib,build, ...etc.)
+	yarn clean
```

Comment:
How about removing the respective entries from the package json files?

Reply:
They are embedded recursively, as each sub-package specifies what is its own cleanup action. We'd have to spec out the cleanup actions for subpackages here, or start using embedded Makefiles as well.